cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -93,15 +101,27 @@ func trackChunks(testData []byte) error {
 		}
 		log.Trace("rpc call ok")
 		count := 0
-		for _, info := range hasInfo {
-			if !info.Has {
-				count++
-				log.Error("Host does not have chunk", "host", httpHost, "chunk", info.Addr)
+		for i, info := range hasInfo {
+			if i == 0 {
```

Comment:
This is not needed, I just wanted to make sure that we get a sorted list back. Once this triggers at least once, and I confirm this is the case, will remove it.